docs/fix-cves.rst

```diff
@@ -0,0 +1,50 @@
+fix-cves
+========
+
+.. argparse::
+   :module: pubtools._pulp.tasks.fix_cves
+   :func: doc_parser
+   :prog: pubtools-pulp-fix-cves
+
+
+Example
+.......
+
+A typical invocation of fix-cves would look like this:
+
+.. code-block::
+
+  pubtools-pulp-fix-cves \
+    --pulp-url https://pulp.example.com/ \
+    --pulp-user admin \
+    --pulp-password XXXXX \
+    --advisory RHXA-123:45 \
+    --cves CVE-123,CVE-345
+
+Mentioned CVEs will be updated in the advisory and uploaded
+to the nochannel repo. The repos this advisory is a part will
```

Comment:
Could you please drop the references to nochannel repo in the docs?

Reply:
Done